org.opent2t.onboarding.getverahub/js/thingOnboarding.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var https = require('https');
+var cryptojs_sha1 = require('crypto-js');
+var request = require('request-promise');
+
+class Onboarding {
+
+    ///
+    /// gets the session token for the provided server url
+    ///
+    getSessionToken(url) {
+        return this.makeVeraRequest(url, this.veraSessionTokenPath, false, false, this.mmsInfo, true);
+    }
+
+    ///
+    /// gets the mms info for the user account
+    ///
+    getMmsAuthInfo() {
+        var path = '/autha/auth/username/' + this.username + '?SHA1Password=' + this.sha1Password + '&PK_Oem=1';
```

Comment:
+1 on standardizing; I've started to adopt the template string as much as I can as I update/modify existing code